e2e/test/scenarios/metrics/metrics-editing.cy.spec.js

```diff
@@ -142,7 +142,7 @@ describe("scenarios > metrics > editing", () => {
       popover().findByText("Edit metric definition").click();
       addBreakout({ tableName: "Product", columnName: "Created At" });
       updateMetric();
-      verifyLineAreaBarChart({ xAxis: "Created At", yAxis: "Count" });
+      verifyLineAreaBarChart({ xAxis: "Product → Created At", yAxis: "Count" });
```

Comment:
For some reason setting the default display type fixes the column name in viz. You can see it is correct by looking at line 143.